spec/serializers/provider_serializer_spec.rb

```diff
@@ -84,4 +84,18 @@
 
     it { should eq provider.ucas_preferences.send_application_alerts_before_type_cast }
   end
+
+  describe 'contacts' do
+    let(:provider) do
+      create :provider,
+             contacts: [utt]
+    end
+    let(:utt) { create(:contact, type: 'utt') }
```

Comment:
made the trait use the trait

`create(:contact, :utt_contact)`

Reply:
Done 👍